docker/Dockerfile

```diff
@@ -17,8 +17,9 @@ LABEL maintainer="Presto community <https://prestosql.io/community.html>"
 ENV JAVA_HOME /usr/lib/jvm/java-11
 RUN \
     set -xeu && \
+    yum -y -q install https://cdn.azul.com/zulu/bin/zulu-repo-1.0.0-1.noarch.rpm && \
```

Comment:
Is this documented somewhere?
Is this URL expected to not change frequently?

Reply:
>Is this documented somewhere? 

It is documented [here](https://docs.azul.com/zulu/zuludocs/ZuluUserGuide/PrepareZuluPlatform/AttachYumRepositoryRHEL-SLES-OracleLinuxSys.htm).

>Is this URL expected to not change frequently?

This is definitely a fair question. My assumption would be no because the url is in their documentation. This has been the url since 2020-06-30 as seen in azul's CDN [here](https://cdn.azul.com/zulu/bin/)..